packages/mapviewer/src/api/layers/WMSCapabilitiesParser.class.js

```diff
@@ -431,8 +431,41 @@ export default class WMSCapabilitiesParser {
         return [new LayerAttribution(title, url)]
     }
 
+    _constructLegendFromGetLegendGraphic(getLegendGraphicObject, layerId) {
+        return (
+            getLegendGraphicObject.DCPType[0].HTTP.Get.OnlineResource +
+            `SERVICE=WMS` +
+            `&REQUEST=GetLegendGraphic` +
+            `&VERSION=${this.version}` +
+            `&FORMAT=${getLegendGraphicObject.Format[0]}` +
+            `&LAYER=${layerId}` +
+            `&SLD_VERSION=1.1.0`
```

Comment:
you should maybe switch to use \` string building pattern overall (instead of concat)